adapters/src/main/java/com/squareup/moshi/adapters/EnumJsonAdapter.java

```diff
@@ -15,13 +15,10 @@
  */
 package com.squareup.moshi.adapters;
 
-import com.squareup.moshi.Json;
-import com.squareup.moshi.JsonAdapter;
-import com.squareup.moshi.JsonDataException;
-import com.squareup.moshi.JsonReader;
-import com.squareup.moshi.JsonWriter;
+import com.squareup.moshi.*;
```

Comment:
While waiting for review, you should fix these star imports

Reply:
done. sorry about that!